shared/genesis.json

```diff
@@ -0,0 +1,45 @@
+{
+    "config": {
+        "chainId": 1,
+        "homesteadBlock": 0,
+        "eip150Block": 0,
+        "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
+        "eip155Block": 0,
+        "eip158Block": 0,
+        "byzantiumBlock": 0,
+        "constantinopleBlock": 0,
+        "petersburgBlock": 0,
+        "istanbulBlock": 0,
+        "muirGlacierBlock": 0,
+        "berlinBlock": 0,
+        "londonBlock": 0,
+        "shanghaiTime": SHANGHAI_TIME,
+        "shardingForkTime": SHARDING_FORK_TIME,
+        "clique": {
+            "period": 5,
+            "epoch": 30000
+        },
+        "terminalTotalDifficulty": 2,
+        "terminalTotalDifficultyPassed": true
```

Comment:
this should be `false` unless you're starting from post-merge genesis.

Reply:
I'm assuming you are generating a post-merge genesis given the bellatrix fork epoch in the chain config.